app/angular/conceptual/conceptual.js

```diff
@@ -107,7 +107,7 @@ const controller = function (ModelAPI, $stateParams, $rootScope, $timeout, $uibM
 			template: '<duplicate-model-modal suggested-name="$ctrl.suggestedName" close="$close(result)" dismiss="$dismiss(reason)"></duplicate-model-modal>',
 			controller: function () {
 				const $ctrl = this;
-				$ctrl.suggestedName = `${model.name} (cópia)`;
+				$ctrl.suggestedName = $filter('translate')('X (copy)', { name: model.name });
```

Comment:
```suggestion
				$ctrl.suggestedName = $filter('translate')('modelName (copy)', { name: model.name });
```

I think it is more descriptive if we use a meaningful word that describes the data placed there instead of using `X`.